### DISCUSSION TITLE
Sync table with type 'datetime' on MySql via ASP.NET Core Web Proxy

### DISCUSSION THREAD

**Discussion by debugjan** [Mimetis/Dotmim.Sync#498] [Q&A] [answered]
Hello,  I've a problem:
i try to sync between sqlite db (Client) <-> 10.5.8-MariaDB (Server, ASP.NET Core Web Proxy) .
I have the following  table:

```
CREATE TABLE `performances` (
[...]
	`Timestamp` DATETIME(6) NOT NULL,
[...]
```

When i sync this (with Dotmim.Sync.Web.Client), i get an exception: `The Column Timestamp of type datetime from provider MySqlSyncProvider, Dotmim.Sync.MySql.MySqlSyncProvider is not currently supported.`

I saw that in some examples (with sqlserver) the type datetime is used.

do I have to change the data type or is there another solution  ?

Thanks in advance

**Comment by Mimetis** [comment, marked as answer]
The message is not explicit enough, I guess.
The problem here is not the column type but the **column name**.
`Timestamp` as name is not allowed. 
If you can change it to `DateTimestamp`, it will work, I guess.

Thanks for the feedback, I will improve the error message for the naming conflict exception.
